yarn.lock

```diff
@@ -3705,19 +4216,6 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@hyperledger/indy-vdr-react-native@patch:@hyperledger/indy-vdr-react-native@npm:0.1.0#./.yarn/patches/@hyperledger-indy-vdr-react-native-npm-0.1.0-10c696b320.patch::locator=bc-wallet-mobile%40workspace%3A.":
```

Comment:
seems like the patch is deleted here

Reply:
it's been added back now after I removed the caret from the resolution block